Zapper-fi/studio
# feat(aurigami): Implement Aurigami

## Description

Integrate Aurigami with Zapper studio.

Aurigami is the native money market on Aurora.

## Checklist


## How to test?

Here is list of Aurigami's endpoints in Zapper studio:
- Fetch supply: `/apps/aurigami/tokens?groupIds[]=supply&network=aurora`
- Fetch borrow position: /`apps/aurigami/positions?groupIds[]=borrow&network=aurora`
- Fetch TVL: `/apps/aurigami/tvl?network=aurora`
- Fetch user balances: `/apps/aurigami/balances?addresses[]=<ADDRESSES>&network=aurora`


To test, you can compare the numbers with [Aurigami App](https://app.aurigami.finance/), or check directly with our contract:
- [Comptroller](https://aurorascan.dev/address/0x817af6cfAF35BdC1A634d6cC94eE9e4c68369Aeb)
- AuTokens:
  - [auUSDC](https://aurorascan.dev/address/0x4f0d864b1ABf4B701799a0b30b57A22dFEB5917b)
  - [auETH](https://aurorascan.dev/address/0xca9511B610bA5fc7E311FDeF9cE16050eE4449E9)
  - [auWBTC](https://aurorascan.dev/address/0xCFb6b0498cb7555e7e21502E0F449bf28760Adbb)
  - [auUSDT](https://aurorascan.dev/address/0xaD5A2437Ff55ed7A8Cad3b797b3eC7c5a19B1c54)
  - [auDAI](https://aurorascan.dev/address/0xCE4166363E3a584DAc84A47bCD3414B43EfCDd1c)
  - [auWNEAR](https://aurorascan.dev/address/0xaE4fac24dCdAE0132C6d04f564dCf059616E9423)
  - [auSTNEAR](https://aurorascan.dev/address/0x3195949f267702723bc614cAE037cdc8D1E94786)
  - [auAURORA](https://aurorascan.dev/address/0x8888682E24dd4Df7B7Ff2B91fccB575737E433bf)
  - [auTRI](https://aurorascan.dev/address/0x6Ea6C03061bDdCE23d4Ec60B6E6e880c33d24dca)
  - [auPLY](https://aurorascan.dev/address/0xC9011e629c9d0b8B1e4A2091e123fBB87B3A792c)
  - [auUSN](https://aurorascan.dev/address/0x5cCAD065400341db391FD3a4B7F50087B678D7CC)
- [AuriLens](https://aurorascan.dev/address/0xFfdFfBDB966Cb84B50e62d70105f2Dbf2e0A1e70)
- [AuriFairLaunch](https://aurorascan.dev/address/0xC9A848AC73e378516B16E4EeBBa5ef6aFbC0BBc2)

## Note
- The USN token price has not been added to the get price API for Aurora yet. So USN market will not show in all the APIs.
